src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -11,6 +13,12 @@ public class AuthenticationService {
      * Return false in any other cases.
      */
     public boolean login(String email, String password) {
+
+        for (User u: getUsers()) {
+            if (u.getEmail().equals(email) && u.getPassword().equals(password)) {
+                return true;
+            }
+        }
```

Comment:
you should create an object of UserService class
then call method findByEmail on this object to find user

then compare passwords